src/pipeline/rackhd/docker/publish.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash -e
+# Environmental requirement:
+# docker service running and docker have already logged with Rackhd Dockerhub ID,
+#     cmd 'docker login', if not logged then can't push images to dockerhub
```

Comment:
is this comment still valid ? you have docker login in your code already

Reply:
Careful observation, I fixed it.